README.md

```diff
@@ -358,3 +360,99 @@ $ vagrant provision
 ```
 
 Use this command when you want to reprovision (re-run the ansible playbook) the vagrant box, you could use this to rerun the playbook if you are in the development process or you want to change some variables.
+
+## About certificates
```

Comment:
Nicely documented, just a comment for the future, we may need to put some thought if we want to have all this documentation in the README. Currently the whole ansible playbook docs is the README, maybe it would be good to rework it and split into more easy to digest docs and keep the readme short to describe only the project. 
But this is a future problem :)